tests/test_modeling_maskformer.py

```diff
@@ -0,0 +1,421 @@
+# coding=utf-8
+# Copyright 2021 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Testing suite for the PyTorch MaskFormer model. """
+
+import inspect
+import unittest
+
+import numpy as np
+
+from tests.test_modeling_common import floats_tensor
+from transformers import MaskFormerConfig, is_torch_available, is_vision_available
+from transformers.file_utils import cached_property
+from transformers.testing_utils import require_torch, require_vision, slow, torch_device
+
+from .test_configuration_common import ConfigTester
+from .test_modeling_common import ModelTesterMixin
+
+
+if is_torch_available():
+    import torch
+
+    from transformers import (
+        MASKFORMER_PRETRAINED_MODEL_ARCHIVE_LIST,
+        MaskFormerForInstanceSegmentation,
+        MaskFormerModel,
+    )
+    from transformers.models.maskformer.modeling_maskformer import (
+        MaskFormerForInstanceSegmentationOutput,
+        MaskFormerOutput,
+    )
```

Comment:
Thos are only used for type annotations, which could be string and are not useful in a test file, so should be removed.